odata-url-conventions/4 Resource Path.md

```diff
@@ -998,4 +999,41 @@ Content-Type: text/plain
 
 $filter=[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]
 ```
+This POST request would result from submitting the HTML form
+```html
+<form method="post" action="http://host/service/People/$query"
+      enctype="text/plain">
+  <input name="$filter"
+    value='[FirstName,LastName]%20in%20[["John","Doe"],["Jane","Smith"]]'>
+</form>
+```
+:::
+
+For `Content-Type: application/x-www-form-urlencoded`, the request body MUST be
+the value of _output_ after running the
+[`application/x-www-form-urlencoded` serializer](https://url.spec.whatwg.org/#concept-urlencoded-serializer)
+in the [URL Living Standard](#_url), section 5.2, with _tuples_ being the list
+of name/value pairs for the individual query options.
+
+::: example
+Example ##ex: passing multiple system query options in the request body
+```
+POST http://host/service/People/$query
+Content-Type: application/x-www-form-urlencoded
+
+%24filter=LastName+eq+%27P%26G%27&%24select=FirstName,LastName
```

Comment:
`$` need not be encoded, the deserializer from WhatWG URL also accepts it.

Servers MUST use the deserializer from ...

Clients MUST serialize it so that the deserializer can make sense of it, for example by using the serializer described in the URL spec.